src/index.ts

```diff
@@ -22,7 +22,7 @@ import arrify = require('arrify');
 import * as extend from 'extend';
 import * as gax from 'google-gax';
 // eslint-disable-next-line node/no-extraneous-import
-import {ClientReadableStream} from '@grpc/grpc-js';
+import {ClientReadableStream, ClientDuplexStream} from '@grpc/grpc-js';
```

Comment:
curious why we need to use a duplex stream, when you're tailing do we also write to the stream?